bin/build_all

```diff
@@ -213,6 +213,7 @@ then
     build_go() {
         docker_build docker/sawtooth-dev-go docker/ sawtooth-dev-go
         docker_build docker/sawtooth-dev-go docker/ sawtooth-tp_intkey_go
+        docker_build docker/sawtooth-dev-go docker/ sawtooth-tp_noop_go
         docker_run sawtooth-dev-go
```

Comment:
You created a sawtooth-int-tp_noop_go Dockerfile but you never build it. Since you don't have any tests for this TP, there isn't a reason to build this here. If you do, then you need to add a line for building the noop_go in installed mode (which is what runs on Jenkins) See line 247 below.